pages/Login/style.js

```diff
@@ -32,6 +33,8 @@ const input = css({
 const forgotWrapper = css({
   textAlign: 'right',
   fontSize: '0.75rem',
+  marginTop: -variables.gap.big,
```

Comment:
The `Forgot password` link isn't clickable anymore, since it's overlapped by the text input 
![bildschirmfoto 2017-12-08 um 16 41 37](https://user-images.githubusercontent.com/13087792/33772993-c570f2b2-dc36-11e7-8acd-7cd3981715a7.png)

Reply:
Uuu. Ugly mistake. Fixed.